mat/vector.go

```diff
@@ -492,6 +492,9 @@ func (v *VecDense) ColViewOf(m RawMatrixer, j int) {
 	if j >= rm.Cols || j < 0 {
 		panic(ErrColAccess)
 	}
+	if !v.IsZero() && v.n != rm.Rows {
```

Comment:
You need to change the documentation to match the behavior.

Reply:
Ah, good catch!